src/main/java/org/cloudfoundry/promregator/PromregatorApplication.java

```diff
@@ -167,7 +167,7 @@ public InternalMetrics internalMetrics() {
 	
 	@Value("${promregator.endpoint.threads:#{null}}")
 	@Deprecated
-	/**
+	/*
```

Comment:
```suggestion
	/**
```
Then let's make it to a real JavaDoc